.circleci/config.yml

```diff
@@ -470,18 +472,31 @@ workflows:
           filters:
             tags:
               only: /.*/
-
-      - build-load-test:
+      - build-test-container:
+          name: Build Load Test Image
+          image: autopush-load-tests
+          path: load
           filters:
             tags:
               only: /.*/
-
-      - build-integration-test:
+            branches:
+              only: master
```

Comment:
This is a departure from the way we have the PR builds setup currently. We usually build the load-test image in PR builds without deploying, so developers can be signaled before they merge if they have introduced a breaking change. I suggest to remove this filter and perhaps the one for the notification test container as well for the same reason.